code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -215,7 +215,7 @@
 
 /obj/item/reagent_containers/hypospray/medipen/survival
 	name = "survival medipen"
-	desc = "A medipen for surviving in the harshest of environments, heals and protects from environmental hazards. WARNING: Do not inject more than one pen in quick succession."
+	desc = "A medipen for surviving in the harshest of environments, heals and protects from environmental hazards. WARNING: Do not inject more than one pen in quick succession, the Nanotransen Exotic Medical Research Division is not responsible for any side effects caused by usage in untested enviroments like space stations."

```

Comment:
I wonder if, for fluff/lore reasons, you should say it's something gravity related. "Intended for use on planetary bodies. May cause unintended side effects under the effects of zero or artificial gravity"

Then again this is another can of worms with Ice Box, so I don't know.

Reply:
lore on why is written in lavaland extract desc